org.eclipse.scanning.command/src/org/eclipse/scanning/command/factory/PyModelExpresser.java

```diff
@@ -46,7 +47,39 @@ String pyExpress(T model, boolean verbose) throws Exception {
 	 * @param verbose
 	 * @return py string
 	 */
-	String pyExpress(T model, Collection<IROI> rois, boolean verbose) throws Exception {
+	String pyExpress(T model, Collection<IROI> rois, @SuppressWarnings("unused") boolean verbose) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)